<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpisodeSelector</class>
 <widget class="QDialog" name="EpisodeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listEpisodes">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>91</y>
     <width>371</width>
     <height>451</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSeasons">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All seasons</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioSeason">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Select entire season(s)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioEpisode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Select individual episode(s)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EpisodeSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EpisodeSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listEpisodes</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>radioEpisode</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboSeasons</sender>
   <signal>activated(int)</signal>
   <receiver>radioSeason</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
